README.md

```diff
@@ -1,17 +1,64 @@
+
+## Related Links
+
+[Learn More about .NET MAUI Range Selector](https://www.syncfusion.com/maui-controls/maui-range-selector?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples)
+
+[Documentation](https://help.syncfusion.com/maui/range-selector/getting-started?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-range-selector?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples)
+
+[Online example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/Sliders/SampleBrowser.Maui.Sliders/Samples/RangeSelector?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples)
+
+[Knowledge base](https://support.syncfusion.com/kb?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples)
+
+## About Syncfusion .NET MAUI Controls
+Syncfusion's [.NET MAUI UI Controls](https://www.syncfusion.com/maui-controls/maui-range-selector?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples) library is the only suite that you will ever need to build an application since it contains over 40 high-performance, lightweight, modular, and responsive UI controls in a single package. In addition to Range Selector, we provide popular .NET MAUI Controls such as [DataGrid](https://www.syncfusion.com/maui-controls/maui-datagrid?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples), [Charts](https://www.syncfusion.com/maui-controls/maui-cartesian-charts?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples), [Scheduler](https://www.syncfusion.com/maui-controls/maui-scheduler?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples), [ListView](https://www.syncfusion.com/maui-controls/maui-listview?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples), and [Excel Library](https://www.syncfusion.com/document-processing/excel-framework/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-range-selector-github-samples).

```

Comment:
check this link 
<img width="819" alt="image" src="https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-range-selector/assets/61148953/dd95ab6b-bd77-4ab7-a688-0042df633b6d">

Reply:
changed link